src/main/java/Event.java

```diff
@@ -0,0 +1,20 @@
+public class Event extends Task {
```

Comment:
Same with Deadline, it's better for the name to reflect that it inherits elements from the superclass, a better name can be something like EventClass